src/components/common/PostalCode/postal-code.scss

```diff
@@ -0,0 +1,50 @@
+@import "src/styles/scaffold";
+
+.postal-code-input {
+  // Lighthouse
```

Comment:
I did't get it. Is there a issue with Lighthouse that we are trying to "solve" here?

Reply:
Oh, I think it was a automatic comment 🤔 
I'll remove it.

Reply:
I think we can ignore this for now since she just copied the existing file. We should clean this up when working on the input inside the regionalization modal.

https://github.com/vtex-sites/base.store/pull/424#discussion_r840126865